docs/concepts/assets/accounts.md

```diff
@@ -62,6 +62,11 @@ Note that this happens instantaneously, so you will likely never observe a curre
 [Concept: Mark-to-market settlement](../trading-on-vega/settlement.md#mark-to-market-settlement) 
 :::
 
+### Holding accounts
+Holding accounts, like settlement accounts, temporarily holds assets to be distributed based on the outcomes of trades and market movements. However, holding accounts are specific to spot markets.
```

Comment:
Not sure how I feel about this line, don't think it adds much and possibly just makes a confusing link to settlement accounts?